app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -53,7 +53,7 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
-        if (datum != null) {
```

Comment:
It is crashing again after doing these changes.

if (datum != null) {
            holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
            if( datum.getTitle() == null || datum.getTitle().isEmpty()) {
                holder.titleTextView.setVisibility(View.GONE);
            } else {
                holder.titleTextView.setVisibility(View.VISIBLE);
                holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
            }